apps/api/src/app/groups/groups.service.ts

```diff
@@ -105,8 +110,17 @@ export class GroupsService {
             )
         }
 
-        group.description = description
-        group.treeDepth = treeDepth
+        if (description) {
```

Comment:
Why that `if`? Without any `if`, if `description` was `undefined` the result wouldn't change. It would just be an empty assignment

Reply:
Wouldn't that clear the existing description value?
What if FE only send the changed values? The API is PUT and not PATCH so ideally it should send all old values as well, but I was thinking of making it more safe.

Reply:
Got it! Makes sense then yea